docu/Concepts/BusinessRequirements/RegelnDerGeldschoepfung.md

```diff
@@ -1,3 +1,22 @@
 # Regeln der Geldschöpfung
 
-*Hier werden die fachlichen Regeln der 3-fachen Schöpfung beschrieben*
+Nach der Gradido-Philosophie erfolgt pro Monat der Prozess der Dreifachen-Geldschöpfung. Das bedeutet, dass in einer Gradido-Community pro Mitglied, das eine *natürliche Person* ist, eine maximale Summe von 3.000 GDD geschöpft wird. Davon kann das Mitglied 1.000 GDD als *Aktives Grundeinkommen* durch Community nützige Leistungen auf sein persönliches AGE-Konto erhalten. 1.000 GDD werden auf das Gemeinwohlkonto der Community gutgeschrieben und weitere 1.000 GDD fließen auf das AUF-Konto der Community.
+
+Um diese Schritte des Geldschöpfungsprozesses zu gewährleisten bedarf es einiger Vorraussetzungen, die im folgenden beschrieben werden.
+
+# Vorraussetzungen
+
+## Drei Kontenmodell
+
+## Verteilungsschlüssel
+
+## Tätigkeitsliste für Aktives Grundeinkommen
+
+Die Tätigkeitsliste für das AGE dient als Grundlage der gemeinnützigen Leistungen, die ein Mitglied sich anrechnen lassen kann. Das heißt eine Community muss eine Sammlung von Tätigkeiten erfassen, die von den Community-Mitgliedern als gemeinnützig empfunden und bestätigt werden. D
+
+Das heißt die Community muss unter den Mitgliedern eine Liste erarbeiten, die alle Tätigkeiten enthält, aus denen sich ein Mitglied dann eine oder mehrere auswählen kann, um sich sein Aktives Grundeinkommen damit zu decken. Die einzelnen Tätigkeiten sollen auch fachlich strukturierbar sein z.B. Kunst, Soziales, Gesundheit, Produktion, etc. . Die Menge und Definition der einzelnen Tätigkeiten und Strukturen unterliegt einer stetigen Anpassung nach den Bedürfnissen der Community-Mitglieder, um den natürlichen Veränderungen des miteinander Lebens gerecht werden zu können. Ob zu einer Tätigkeitsbeschreibung auch gleich eine Wertigkeit definiert werden soll, ist noch unklar. Man kann aber sicherlich sagen, dass manche Tätigkeiten dem Gemeinwohl dienlicher sind als andere. Aber auch das ist wiederum eine Ansichtsache und muss unter den Community-Mitgliedern vereinbart werden.
```

Comment:
Ich denke die Liste sollte sich aus den Tätigkeiten der anerkannten Leistungen zusammensetzen, folglich vollautomatisch entstehen.
Die Zuordnung jeder Leistung zu einer Kategorie halte ich allerdings für sehr gut, da es einen Vergleich und eine Bewertung von Communities zulässt "Das ist eine Handwerker Community" "Das sind ja nur Baumschmuser"

Reply:
Ich denke im ersten Schritt können wir aus den bisher aufgelaufenen Tätigkeiten während der Transformationsphase eine Art Default-Liste erzeugen, die dann von den Communities weiter ergänzt/verändert, etc. werden. 
Ob wir hier evtl. schon Schrittweise in den jetzigen Adminbereich Mechanismen einbauen sollten, um eine Gliederung und feste Vorschlagstexte der Tätigkeitsbezeichnungen zu gewinnen, sollten wir im Hinterkopf behalten.